packages/app-project/src/components/ProjectHeader/ProjectHeader.js

```diff
@@ -23,7 +23,7 @@ function ProjectHeader (props) {
   const { className, navLinks, projectHomeLink, size, title } = props
 
   return (
-    <StyledBox className={className}>
+    <StyledBox as='header' className={className} role='presentation'>
```

Comment:
I'd take off the presentation role here, which removes the element completely from the page while browsing, and make this the page header.